typeDefs.js

```diff
@@ -435,6 +448,12 @@ const typeDefs = gql`
     addContact(contact: contactInput!): ContactResult
     deleteContact(id: ID!): ContactResult
     updateContact(id: ID!, contact: contactInput!): ContactResult
+
+    "EmailChat Mutation"
+    addEmailQuestion(question: EmailQuestionInput!): EmailQuestion
+    deleteEmailQuestion(id: ID!): EmailQuestionResult
+    spamQuestion(questionId: ID!): EmailQuestionResult
+    emailChatAnswer(questionId: ID!, text: String!): EmailQuestionResult
```

Comment:
Please, rename it to the `answerEmailQuestion`, mutation should start from the verb

Reply:
done